egs2/TEMPLATE/asr1/db.sh

```diff
@@ -10,6 +10,7 @@ AISHELL4=downloads
 ALFFA=downloads
 AMERICASNLP22=downloads
 AN4=downloads
+APHASIABANK=/ocean/projects/cis210027p/shared/corpora/AphasiaBank/English
```

Comment:
TODO: replace absolute path

Reply:
Maybe you should note somewhere (e.g. data.sh or README in the recipe) that this is only concerned with English.